readme.txt

```diff
@@ -53,17 +51,17 @@ You can get started with ImageEngine easily with a [free, no credit card require
 * **Standard** - $99/month. Up to **250 GB** per month. Includes 3 custom domains (CNAME) with HTTPS support. Priority onboarding support.
 * **Pro** - pricing scales with usage volume.  WAF with DDoS protection. Dedicated edge servers available. Ticketed enterprise support. [Contact us](https://imageengine.io/contact?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine).
 
-= Features =
+= Features & BENEFITS=
```

Comment:
Is `BENEFITS` supposed to be in shouty-caps?

Reply:
probably not... fixing